lib/TwoFactorAuth.php

```diff
@@ -228,21 +218,21 @@ private function codeEquals(string $safe, string $user): bool
 
     private function getTime(?int $time = null): int
     {
-        return ($time === null) ? $this->getTimeProvider()->getTime() : $time;
+        return $time ?? $this->getTimeProvider()->getTime();
     }
 
     private function getTimeSlice(?int $time = null, int $offset = 0): int
     {
-        return (int) floor($time / $this->period) + ($offset * $this->period);
+        return (int)floor($time / $this->period) + ($offset * $this->period);
```

Comment:
I find a space after the type coercion much more readable

Reply:
Another PHP-CS-Fixer rule.